### PR TITLE
Follow-up: Installation date "today" not correct (EXPOSUREAPP-7482)

### PR DESCRIPTION
**Pull request by marcauberer**
Addresses [EXPOSUREAPP-7482](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7482)

Change text `seit heute installiert` to `wurde heute installiert`. Same for `gestern`.

### CODE REVIEW

**Review by NataliaLemmerth — DISMISSED** [review]
UA approved

File: Corona-Warn-App/src/main/res/layout/home_submission_status_card_unregistered.xml [hunk 1]
```diff
@@ -39,7 +39,7 @@
 
         <ImageView
             android:id="@+id/icon"
-            android:layout_width="wrap_content"
+            android:layout_width="95dp"
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/res/layout/home_submission_status_card_unregistered.xml:42]: why did you change this?

Comment (marcauberer) [reply Corona-Warn-App/src/main/res/layout/home_submission_status_card_unregistered.xml]: Oh that was a unintended change. Just reverted it. Thanks for mentioning 👍🏼

**Review by mtwalli — APPROVED** [review]

**Review by ralfgehrer — APPROVED** [review]
